qna/de-webshop-order-sandbox.md

```diff
@@ -0,0 +1,23 @@
+## Question
+
+Ich habe im fiskaltrust.Portal auf der Sandbox eine TSE bestellt, aber erhalte keine Lieferung. Was ist da schiefgelaufen?
+
+## Question
+
+Im Webshop in der Sandbox sehe ich andere Produkte als im Produktions-Portal, teilweise auch mit anderen Preisen. Welche Informationen und Produkte sind korrekt?
+
+## Metadata tags
+
+lang-de, market-de, market-at, middleware, PosCreator, POSDealer, PosOperator, Consultant
+
+## Answer
+
+Die Sandbox-Umgebung dient Kassenherstellern als Integrationsumgebung, und wird auch von fiskaltrust als Teil des Release Prozesses verwendet.
+
+Das bedeutet, dass wir hier neue Entwicklungen unter möglichst produktionsnahen Bedingungen testen. Im Zuge dessen werden z.B. Test-Produkte, Daten oder Informationen erzeugt, denen nie bestimmt ist, in Produktion zu gelangen.
+
+Webshop Bestellungen von Hardware-Produkten werden in der Sandbox realitätsnah abgebildet, inklusive Auftragsbestätigung und Rechnungsversand, wobei die Rechnungen um den Vermerk ergänzt werden, dass es sich um eine Test-Rechnung aus der Sandbox-Umgebung handelt. Im Gegenzug zum Produktiv-Portal wird im Hintergrund der Bestell-Auftrag aus der Sandbox nicht an unseren Logistikpartner übermittelt.
+
+In der Sandbox kommt daher keine rechtsverbindliche Beauftragung und auch kein Versand von Hardware zustande. "Echte" Bestellungen können ausschliesslich über den Webshop in Produktion durchgeführt werden.
+
+Auch die Bestellung digitaler Produkte in der Sandbox, wie z.B. ein Sorglos-Paket, hat nur in der Sandbox Wirkung.
```

Comment:
Dies gilt für alle Produkte, auch digitaler, in der Sandbox. Eine Verwendung der in der Sandbox zur Verfügung gestellten Produkte für produktive Systeme ist möglich. Diese Produkte sind nur für Test- bzw. Lernzwecke geeignet und stellen keine rechtsgültige Fiskalisierung von Belegen zur Verfügung.

Reply:
scope dieser FAQ ist dass es immer wieder Kunden gibt die in der Sandbox im Webshop Produkte bestellen und dann bezüglich der Lieferung nachfragen. ich denke die antwort deckt das so weit recht gut ab.